test/inproc.js

```diff
@@ -15,6 +15,8 @@ test('inproc socket pub sub', function (t) {
     var addr = 'inproc://pubsub';
     var msg = 'hello world';
 
+    sub.chan('');
```

Comment:
> can we remove all references to the '' chan from tests that don't test filtering on a sub socket?

that would be useful. maybe if we run a setTimeout or Interval after sub socket is created that checks for the existence of any length on `this.channel`. 

If we did something like that it would make more sense to only accept arrays for topics, since we could just check length on that property.

well if you have a better implementation idea in mind I'm all ears, but the main thing is that once we've subscribed to `''`, there's no point in registering subsequent subscription topics, since these plus all others would get filtered into the `message` event callback